components/gitpod-protocol/src/messaging/error.ts

```diff
@@ -41,6 +41,9 @@ export namespace ErrorCodes {
     // 450 Payment error
     export const PAYMENT_ERROR = 450;
 
+    // 451 Out of credits
+    export const PAYMENT_OUT_OF_CREDITS = 451;
```

Comment:
:cloud: Would it make sense to clearly distinguish from the other payment error, maybe by adding a "usage-based" here? :thinking:

Reply:
I think I dropped a commit to rename this to `PAYMENT_SPENDING_LIMIT_REACHED` that's a more clear signal (in code,) but in terms of error codes that's not relevant, and also the payload is where we can encode more specific details.